libs/clients/work-machines/src/lib/workMachines.service.ts

```diff
@@ -4,33 +4,120 @@ import {
   ApiMachinesGetRequest,
   ExcelRequest,
   GetMachineRequest,
+  MachineCategoryApi,
+  MachineFriendlyDto,
   MachineHateoasDto,
+  MachineOwnerChangeApi,
   MachinesApi,
   MachinesDocumentApi,
   MachinesFriendlyHateaosDto,
 } from '../../gen/fetch'
+import { CustomMachineApi } from './providers'
+import {
+  MachineDto,
+  ChangeMachineOwner,
+  ConfirmOwnerChange,
+} from './workMachines.types'
+import {
+  apiChangeMachineOwnerToApiRequest,
+  confirmChangeToApiRequest,
+} from './workMachines.utils'
 
 @Injectable()
 export class WorkMachinesClientService {
   constructor(
     private readonly machinesApi: MachinesApi,
     private readonly docApi: MachinesDocumentApi,
+    private readonly machineApi: CustomMachineApi,
+    private readonly machineOwnerChangeApi: MachineOwnerChangeApi,
+    private readonly machineCategoryApi: MachineCategoryApi,
   ) {}
 
-  private apiWithAuth = (user: User) =>
+  private machinesApiWithAuth = (user: User) =>
     this.machinesApi.withMiddleware(new AuthMiddleware(user as Auth))
 
+  private machineApiWithAuth(auth: Auth) {
+    return this.machineApi.withMiddleware(new AuthMiddleware(auth))
+  }
+
+  private machineOwnerChangeApiWithAuth(auth: Auth) {
+    return this.machineOwnerChangeApi.withMiddleware(new AuthMiddleware(auth))
+  }
+
+  private machineCategoryApiWithAuth(auth: Auth) {
+    return this.machineCategoryApi.withMiddleware(new AuthMiddleware(auth))
+  }
   getWorkMachines = (
     user: User,
     input: ApiMachinesGetRequest,
   ): Promise<MachinesFriendlyHateaosDto> =>
-    this.apiWithAuth(user).apiMachinesGet(input)
+    this.machinesApiWithAuth(user).apiMachinesGet(input)
 
   getWorkMachineById = (
     user: User,
     input: GetMachineRequest,
-  ): Promise<MachineHateoasDto> => this.apiWithAuth(user).getMachine(input)
+  ): Promise<MachineHateoasDto> =>
+    this.machinesApiWithAuth(user).getMachine(input)
 
   getDocuments = (user: User, input: ExcelRequest): Promise<Blob> =>
     this.docApi.withMiddleware(new AuthMiddleware(user as Auth)).excel(input)
+
+  public async getMachines(auth: User): Promise<MachineFriendlyDto[]> {
```

Comment:
The default visibility is `public`

```suggestion
 async getMachines(auth: User): Promise<MachineFriendlyDto[]> {
```